bin/functional-tests/conftest.py

```diff
@@ -10,9 +10,9 @@
     print("NAMESPACE env var is not present, using 'astronomer' namespace")
     namespace = "astronomer"
 
-if not (release_name := getenv("RELEASE_NAME")):
+if not (release_name := getenv("release_NAME")):
```

Comment:
Good catch! This was just a quick late-night start. I should've marked it as not ready for review yet since I was really just looking to see if CI passed. I'll circle back and give it a closer look when I have time.

Reply:
Or... I guess I'm mis-remembering since I specifically asked for review above, haha